examples/cifar10_resnet.py

```diff
@@ -130,26 +164,31 @@
 # Instantiate and compile model.
 model = Model(inputs=inputs, outputs=outputs)
 model.compile(loss='categorical_crossentropy',
-              optimizer=Adam(),
+              optimizer=Adam(lr=lr_schedule(0)),
               metrics=['accuracy'])
 model.summary()
 
+if version == 2:
+    print("ResNet v2")
```

Comment:
Nit: use `'` for consistency with rest of file

Reply:
Revised using ''.